apps/dashboard/src/components/primitives/control-input/variable-plugin/variable-pill-widget.ts

```diff
@@ -60,7 +70,7 @@ export class VariablePillWidget extends WidgetType {
       fontFamily: 'var(--font-code)',
       display: 'inline-flex',
       alignItems: 'center',
-      height: '100%',
+      height: '16px',
```

Comment:
the pill looks smaller now than it used to be
![Screenshot 2025-04-16 at 12 48 30](https://github.com/user-attachments/assets/76e9997e-2385-484f-8fa9-776d1a38f8ef)